Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -394,10 +439,38 @@ private class LoginCursorDataSource: NSObject, UITableViewDataSource {
 
     var cursor: Cursor<Login>?
 
+    var loginFaviconMap = [Login: [Favicon]]()
+
     func loginAtIndexPath(indexPath: NSIndexPath) -> Login {
         return loginsForSection(indexPath.section)[indexPath.row]
     }
 
+    func indexPathForLogin(login: Login) -> NSIndexPath? {
+        guard let baseDomain = login.hostname.asURL?.baseDomain() else {
```

Comment:
Does this mean we don't show non-domain entries at all? Is that documented somewhere?